### ISSUE TITLE
Investigate behaviour and styling of Readonly Input Elements

### ISSUE THREAD

**Issue by ryanthemanuel** [cerner/terra-core#3248]
# Bug Report

## Description

As of now there are no defined standards for the behaviour/styling of " Readonly Input Elements" when defined using the **html readonly** attribute.

## Expected Behavior

The readonly behaviour and styling for input elements should be consistent throughout all terra components.

## @ Mentions

@neilpfeiffer @jeremyfuksa

**Comment by ryanthemanuel** [comment]
We will need to spike on this from a UX standpoint before we are ready to start working on this.
